oauth2client/flask_util.py

```diff
@@ -320,10 +336,11 @@ def _make_flow(self, return_url=None, **kwargs):
         kw = self.flow_kwargs.copy()
         kw.update(kwargs)
 
-        extra_scopes = util.scopes_to_string(kw.pop('scopes', ''))
-        scopes = ' '.join([util.scopes_to_string(self.scopes), extra_scopes])
+        extra_scopes = kw.pop('scopes', [])
+        scopes = util.scopes_to_string(
```

Comment:
Hahaha. I don't think the call is needed. I'll remove it and see if anything blows up.